Logger.Tests/LogFactoryTests.cs

```diff
@@ -5,7 +5,44 @@ namespace Logger.Tests
 {
     [TestClass]
     public class LogFactoryTests
-    {
-        
+    {
+
+        [TestMethod]
+        public void Logger_Returns_Null()
+        {
+            LogFactory logFactory = new LogFactory();
+
+            BaseLogger testLogger = logFactory.CreateLogger(nameof(testLogger));

```

Comment:
```suggestion
            BaseLogger? testLogger = logFactory.CreateLogger(nameof(testLogger));

```
Nullability should be enabled.